docs/scarpet/api/Auxiliary.md

```diff
@@ -268,6 +268,9 @@ Example usages:
   // not a problem in apps
 </pre>
 
+### `item_display_name(item, hasbracket?)`
+ returns the `display_name` of the item as a Text Value. if hasbracket is true, there will be a bracket around it.
+
```

Comment:
IMO you should mention that `item` can be either an item triplet of `[item_name, count, nbt]`, or an item name.